scripts/install

```diff
@@ -101,61 +29,36 @@ ynh_script_progression --message="Installing $app with Composer..." --weight=5
 
 ynh_install_composer --install_args="--ignore-platform-reqs" 2>&1
 
-pushd $final_path
-	patch --strip=1 < $YNH_APP_BASEDIR/sources/app-array_key_exists.patch
-popd 
+pushd "$install_dir"
+    patch --strip=1 < "$YNH_APP_BASEDIR/sources/app-array_key_exists.patch"
+popd
 
```

Comment:
Bit confused by this .. patches are supposed to already be handled by ynh_setup_source ..

For some reason this doesnt make the install crash, but it seems to make the upgrade crash because the patch was already applied ..

Reply:
Aren't they supposed to be in a patches directory ?

Also I was wondering if the patch is applied AFTER the composer build on purpose...

Reply:
Apparently it was done on purpose here https://github.com/YunoHost-Apps/quizzes_ynh/pull/7/commits/2f84edd47addd8c61c69e93c5e9568b2df2f4c8d but no real explanation why x_x ...